ansible_collections/arista/avd/roles/eos_designs/templates/inband_management/parent-logic.j2

```diff
@@ -4,11 +4,12 @@
 
 {# Add underlay links defined on other devices pointing to this device #}
 {% for fabric_switch in groups[fabric_name] | arista.avd.natural_sort %}
-{%     if inventory_hostname in hostvars[fabric_switch].switch.inband_management_parents | arista.avd.default([]) %}
+{%     set fabric_switch_facts = hostvars[fabric_switch].switch | arista.avd.default %}
+{%     if inventory_hostname in fabric_switch_facts.inband_management_parents | arista.avd.default([]) %}
 {%         set inband_management_data.role = 'parent' %}
-{%         if hostvars[fabric_switch].switch.inband_management_subnet not in inband_management_data.subnets %}
-{%             do inband_management_data.vlans.append(hostvars[fabric_switch].switch.inband_management_vlan) %}
-{%             do inband_management_data.subnets.append(hostvars[fabric_switch].switch.inband_management_subnet) %}
+{%         if fabric_switch_facts.inband_management_subnet not in inband_management_data.subnets %}
+{%             do inband_management_data.vlans.append(fabric_switch_facts.inband_management_vlan) %}
+{%             do inband_management_data.subnets.append(fabric_switch_facts.inband_management_subnet) %}
 {%         endif %}
 {%     endif %}
 {% endfor %}
```

Comment:
Missing new line. Probably this was already present.

Reply:
LGTM!

Reply:
Fixed